src/PixiEditor/Views/Dialogs/CrashReportDialog.xaml

```diff
@@ -29,7 +29,7 @@
             <RowDefinition Height="Auto"/>
             <RowDefinition/>
         </Grid.RowDefinitions>
-        <dial:DialogTitleBar TitleText="PixiEditor has crashed!" CloseCommand="{x:Static SystemCommands.CloseWindowCommand}" />
+        <dial:DialogTitleBar TitleKey="PixiEditor has crashed!" CloseCommand="{x:Static SystemCommands.CloseWindowCommand}" />
```

Comment:
That's not a valid key

Reply:
I didn't change it because crash reports don't have ViewModelMain initialized.

Reply:
Yeah so, fun fact.

Crash Report window crashes because it doesn't have localization initialized